### PR TITLE
"What is Agoric?" copy

### PR DESCRIPTION
**Pull request by dckc**

### CODE REVIEW

File: main/what-is-agoric.md [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+The Agoric blockchain is built on the CometBFT consensus system with
+native Inter-Blockchain Communication Protocol (IBC) support. Smart
+contracts can seamlessly integrate 3rd party libraries, so a contract
+could invoke an operation on Avalanche via Axelar’s GMP the same way
+it does on Osmosis via IBC. Agoric smart contracts are long-lived
```
Comment (dckc) [inline main/what-is-agoric.md:15]: Do we want to include this even though the docs don't include any way for contracts to use IBC?

File: main/what-is-agoric.md [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+time with scheduled, long-term, and periodic actions
+(e.g. subscriptions, mortgages).
 
-Beyond just the JavaScript programming language, the Agoric platform introduces Zoe, a “first of its kind” platform service that enables safe economic composition between smart contracts. Zoe brings to web3 developers on the Agoric platform the kind of modern, pluggable component framework that has dominated web2 development.
+Agoric’s hardened JavaScript makes blockchain programming accessible
+to the 17+ million JavaScript developers around the world. Hardened
+JavaScript provides the safe, stable environment that developers need
+to build, deploy, and operate sophisticated applications, NFTs, and
+DeFi markets. Beyond the JavaScript programming language, the Agoric
+platform introduces Zoe, a “first of its kind” escrow service that
```
Comment (dckc) [inline main/what-is-agoric.md:28]: Why is "first of its kind" quoted?

File: main/what-is-agoric.md (hunk 1 above)
Comment (dckc) [reply main/what-is-agoric.md:15]: @sufyaankhan said yes; we've been public that it's coming

**Review by kbennett2000 — APPROVED** [review]

File: main/what-is-agoric.md (hunk 2 above)
Comment (dckc) [reply main/what-is-agoric.md:28]: I guess I'll just leave it that way.

**Pull request merged**